source/stable-release_iot/stable-release_iot.rst

```diff
@@ -0,0 +1,17 @@
+.. _stable-celadon-iot:
+
+Stable Celadon IoT
+##################
+
+Celadon IoT is an Internet of Things (IoT) overlay on top of Celadon. Celadon IoT, release provides users with an extensive set of features on Comet Lake(CML), Tiger Lake(TGL), and Elkhart Lake(EHL) Intel® platforms with single Baseline/Image.
+
+Celadon IoT offers 100% compliance for Compatibility Test Suite (CTS), Vendor Test Suite (VTS), CTS-on-GSI, GMS Test Suite (GTS), and Security Test Suite (STS) on Comet Lake(CML), Tiger Lake(TGL), and Elkhart Lake(EHL). 
```

Comment:
This is repetitive, either cover here or in feature delta iot that you have already covered.

Reply:
Updated as per the Suggestion: Removed 2nd Bulletin from stable-release_iot.rst file